src/stars.css

```diff
@@ -0,0 +1,29 @@
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-size: 13px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--0 .stars__star:nth-child(-n + 5),
+.stars--1 .stars__star:nth-last-child(-n + 4),
+.stars--2 .stars__star:nth-last-child(-n + 3),
+.stars--3 .stars__star:nth-last-child(-n + 2),
+.stars--4 .stars__star:nth-last-child(-n + 1) {
+  background-image: url(/src/images/star.svg);
```

Comment:
You can just move this property to `.stars_star selector`. It would be then overtwritten by the below selectors with `nth-child` bcs it has bigger specificity.